iox-#743 Update iceperf readme (#1256)
eclipse-iceoryx/iceoryx: iceoryx_examples/iceperf/README.md

## Introduction

> !!! Note

Reviewer: Not sure if a capital letter works with MkDocs, please check it before merging this PR.

```suggestion
!!! note
```

## Expected Output

The measured transmission modes depend on the operating system (e.g. no message queue on MacOS).
The measurements depend on the benchmark parameters and the hardware.

> The following showss an example output with Ubuntu 18.04 on Intel(R) Xeon(R) CPU E3-1505M v5 @ 2.80GHz.

Reviewer 2: ```suggestion
The following shows an example output with Ubuntu 18.04 on Intel(R) Xeon(R) CPU E3-1505M v5 @ 2.80GHz.
```

The `PerfSettings` struct is used to synchronize the settings between the leader and the follower application.

The `PerfTopic` struct is used to share some information during the measurement. It contains `payloadSize`
to specify the payload size used for the current measurement. If it is not possible to transmit the `payloadSize`
with a single data transfer (e.g. OS limit for the payload of a single socket send), the payload is divided

> into several sub-packets. This is indicated with `subPackets`. The `runFlag` is used to shutdown the

Reviewer 2: ```suggestion
into several sub-packets. This is indicated with `subPackets`. The `runFlag` is used to shut down the
```
